examples/nifpga/read_write_array_example.py

```diff
@@ -0,0 +1,126 @@
+r""" Read and Write to indicators, controls, Arrays and FIFO.
+
+The gRPC API is built from the C API. NI-FPGA documentation is installed with the driver at:
+    C:\Program Files (x86)\National Instruments\FPGA Interface C API\capi.chm
+
+Getting Started:
+
+To run this example, you need to provide the correct file path and bitfile signature.
+Additionally, ensure you have a VI that includes arrays and update the references for them below.
+Make sure the R-Series drivers are installed on the system where the gRPC-device server is running.
+You will also need LABVIEW and the LABVIEW FPGA module to configure the VIs.
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Run from command line as we need to pass Resource Name:
+
+Server machine's IP address, port number, and resource name can be passed as separate command line
+arguments.
+  > python read_write_example.py <server_address> <port_number> <resource_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "" as the resource name.
+
+This example attempts to read and write values to an array.
+"""
+
+import sys
+
+import grpc
+import nifpga_pb2 as nifpga_types
+import nifpga_pb2_grpc as grpc_nifpga
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+SESSION_NAME = "NI-FPGA-Session"
+
+RESOURCE = ""
+
+# These values should be configured according to the VI you are using.
+NI_FPGA_EXAMPLE_BITFILE_PATH = "C:\\DemoExample.lvbitx"
+NI_FPGA_EXAMPLE_SIGNATURE = "11F1FF1D11F1FF1F1F111FF11F11F111"
+NiFpga_ExampleVI_ControlArrayI32_Array = 0x10004
```

Comment:
This is the naming convention we use when generating **.h** files for the VI. Although I can change the names for the signature and path variables, we wanted to keep the other variables consistent with the .h file naming convention so that users can easily understand them. The values essentially reference the controls on the VI.
![image](https://github.com/user-attachments/assets/9803a5ca-eed2-42bd-82ad-aa71b83cf5ec)

Reply:
Updated accordingly after internal discussion.